cmake/hsConfig.h.cmake

```diff
@@ -45,6 +45,14 @@ You can contact Cyan Worlds, Inc. by email legal@cyan.com
 
 /* Compiler settings */
 #cmakedefine HAVE_CPUID
+#cmakedefine HS_AVX2
+#cmakedefine HS_AVX
+#cmakedefine HS_SSE42
+#cmakedefine HS_SSE41
+#cmakedefine HS_SSE4
+#cmakedefine HS_SSE3
+#cmakedefine HS_SSE2
+#cmakedefine HS_SSE1
```

Comment:
I realize these are the names we used before, but perhaps `HAVE_<instruction set>` would be simpler / more readable?